src/EnergyPlus/CMakeLists.txt

```diff
@@ -991,11 +992,37 @@ if(BUILD_PACKAGE)
-      install(CODE "execute_process(COMMAND powershell.exe -ExecutionPolicy Bypass -File ${PROJECT_SOURCE_DIR}/scripts/dev/create_shortcut.ps1 -TargetPath \"$<TARGET_FILE:energyplus>\" -ShortcutPath \"$<TARGET_FILE_DIR:energyplus>/EPLaunchPython.lnk\" -Arguments \"auxiliary eplaunch\")" COMPONENT Auxiliary)
-      install(FILES $<TARGET_FILE_DIR:energyplus>/EPLaunchPython.lnk DESTINATION "./" COMPONENT Auxiliary)
+    if(PYTHON_CLI)
+      if(WIN32)
+        # on Windows, with Build Package, and also Link With Python, we can also drop in shortcuts to the new auxiliary CLI
+        add_executable(windows_gui_launcher WIN32 WindowsGuiLauncher.cc)
+        target_link_options(windows_gui_launcher PRIVATE /SUBSYSTEM:WINDOWS)
+        install(TARGETS windows_gui_launcher DESTINATION ./ COMPONENT Auxiliary)
+        # NOTE: The install command COMPONENTS should line up so that they are run at the same packaging step
+        install(CODE "execute_process(COMMAND powershell.exe -ExecutionPolicy Bypass -File ${PROJECT_SOURCE_DIR}/scripts/dev/create_shortcut.ps1 -TargetPath \"$<TARGET_FILE_DIR:energyplus>/windows_gui_launcher.exe\" -ShortcutPath \"$<TARGET_FILE_DIR:energyplus>/EPLaunchPython.lnk\" -Arguments \"eplaunch\")" COMPONENT Auxiliary)
+        install(CODE "execute_process(COMMAND powershell.exe -ExecutionPolicy Bypass -File ${PROJECT_SOURCE_DIR}/scripts/dev/create_shortcut.ps1 -TargetPath \"$<TARGET_FILE_DIR:energyplus>/windows_gui_launcher.exe\" -ShortcutPath \"$<TARGET_FILE_DIR:energyplus>/IDFVersionUpdater.lnk\" -Arguments \"updater\")" COMPONENT Auxiliary)
+        install(FILES $<TARGET_FILE_DIR:energyplus>/EPLaunchPython.lnk DESTINATION "./" COMPONENT Auxiliary)
+        install(FILES $<TARGET_FILE_DIR:energyplus>/IDFVersionUpdater.lnk DESTINATION "./" COMPONENT Auxiliary)
+      elseif(APPLE)
+        # Pre-signed and notarized launcher apps
+        install(DIRECTORY "${PROJECT_SOURCE_DIR}/bin/PythonApps/EPLaunch.app" DESTINATION "." COMPONENT Auxiliary USE_SOURCE_PERMISSIONS)
+        install(DIRECTORY "${PROJECT_SOURCE_DIR}/bin/PythonApps/VersionUpdater.app" DESTINATION "." COMPONENT Auxiliary USE_SOURCE_PERMISSIONS)
+      elseif(LINUX)
+        set(ARGUMENT eplaunch)
+        configure_file(${PROJECT_SOURCE_DIR}/cmake/auxiliary.sh.in ${PROJECT_BINARY_DIR}/EPLaunch)
+        install(FILES ${PROJECT_BINARY_DIR}/EPLaunch
+                DESTINATION "."
+                PERMISSIONS OWNER_EXECUTE OWNER_READ OWNER_WRITE)
+        set(ARGUMENT updater)
+        configure_file(${PROJECT_SOURCE_DIR}/cmake/auxiliary.sh.in ${PROJECT_BINARY_DIR}/VersionUpdater)
+        install(FILES ${PROJECT_BINARY_DIR}/VersionUpdater
+                DESTINATION "."
+                PERMISSIONS OWNER_EXECUTE OWNER_READ OWNER_WRITE)
+        install(
+                FILES ${PROJECT_SOURCE_DIR}/cmake/energyplus.desktop.in
+                DESTINATION share/applications
+        )
```

Comment:
On Linux we now need to create a couple shell scripts to launch EPLaunch and VersionUpdater right in the install root.  On second thought, maybe these aren't needed?  I'll consider removing them.

And we also now install a .desktop template inside install root / share/applications to be customized at install time by the QtIFW scripts.